test/test_image.py

```diff
@@ -94,10 +94,10 @@ def test_decode_jpeg(img_path, pil_mode, mode):
     data = read_file(img_path)
     img_ljpeg = decode_image(data, mode=mode)
 
-    # Permit a small variation on pixel values to account for implementation
-    # differences between Pillow and LibJPEG.
+    # Permit a small variation on pixel values to account for differences
+    # between Pillow's libjpeg and ours.
```

Comment:
Should this be

```suggestion
    # between Pillow's libjpeg-tubro and ours.
```

?